shared/tracker/index.js

```diff
@@ -131,6 +131,7 @@ export default connect(
   },
   (stateProps, dispatchProps, ownProps) => {
     return {
+      proofs: [],
```

Comment:
Hmm I can't seem to repro. Sounds like this might be a bug in how we store the tracker state? Does the rest of the tracker state exists and it's only proof that's missing?